src/base/orgs/View.svelte

```diff
@@ -191,74 +169,73 @@
+            {:then authorized}
+              {#if authorized}
+                <button class="tiny secondary" on:click={setName}>
+                  Set
+                </button>
+              {/if}
+            {/await}
+          </div>
         </div>
-      </div>
 
-      <div class="members">
-        {#await org.getMembers(config)}
-          <Loading center />
-        {:then members}
-          {#each members as address}
-            <div class="member">
-              <div class="member-icon">
-                <Blockies {address} />
+        <div class="members">
+          {#await org.getMembers(config)}
+            <Loading center />
+          {:then members}
+            {#each members as address}
+              {#await Profile.get(address, config)}
+                <Loading small center />
+              {:then profile}
+              <div class="member">
+                <div class="member-icon">
+                  <Avatar source={profile.avatar() ??  address} />
+                </div>
+                <Address {address} compact resolve noAvatar {config} />
               </div>
-              <a href={explorerLink(address, config)} target="_blank" class="member">
-                {utils.formatAddress(address)}
-              </a>
-            </div>
-          {/each}
-        {/await}
-      </div>
-
-      <div class="projects">
-        {#await org.getProjects(config)}
-          <Loading center />
-        {:then projects}
-          {#each projects as project}
-            <div class="project">
-              <Project {project} org={org.address} {config} />
-            </div>
-          {/each}
-        {:catch err}
-          <Message error>
-            <strong>Error: </strong> failed to load projects: {err.message}.
-          </Message>
-        {/await}
-      </div>
+              {/await}
+            {/each}
+          {/await}
+        </div>
+        
+        <div class="projects">
+          {#await org.getProjects(config)}
+            <Loading center />
+          {:then projects}
+            {#each projects as project}
+              <div class="project">
+                <Project {project} org={org.address} {config} />
+              </div>
+            {/each}
+          {:catch err}
+            <Message error>
+              <strong>Error: </strong> failed to load projects: {err.message}.
+            </Message>
+          {/await}
+        </div>
+      {/await}
     </main>
   {:else}
     <Modal subtle>
       <span slot="title">🏜️</span>
       <span slot="body">
-        <p class="highlight"><strong>{address}</strong></p>
+        <p class="highlight"><Address compact {address} {config} /></p>
         <p>Sorry, there is no Org at this address.</p>
         {#if utils.isAddress(address)}
           <p>
             <a href={explorerLink(address, config)} class="link" target="_blank">View in explorer</a>
             <span class="faded">↗</span>
           </p>
-        {/if}
+          {/if}
```

Comment:
Typo?

Reply:
Yeah it seems a wild indentation appeared :man_shrugging: